proprietary/tokens/src/components/utrecht/button.tokens.json

```diff
@@ -0,0 +1,271 @@
+{
+  "utrecht": {
+    "button": {
+      "background-color": {},
```

Comment:
I don't think you need to include empty tokens, they aren't converted into css variables or anything

Reply:
For now I would like to keep it. Just for the references what the possibilities are regarding token settings. I will create a ticket to discuss this.